funcx_container_service/container.py

```diff
@@ -109,7 +110,7 @@ def uncompress_payload(self, payload_path):
                     tar_obj.extractall(self.temp_dir)
             except Exception as e:
                 err_msg = f'untar failed: {e}'
-                self.log_error(err_msg)
```

Comment:
@BenGalewsky with the removal of this call to `log_error`, we lose the functionality that updates the build status on the webservice along with cleanup of the temp dir. If you think that raising an exception here is the correct course of action, we could work the functionality back into a larger try/catch block that performs the status update and cleanup in the catch?

Reply:
Ah yes - I had thought about capturing these in a higher level `try/catch` block, but this makes sense here. Restore the `log_error` calls